models/public/ssd_mobilenet_v1_fpn_coco/ssd_mobilenet_v1_fpn_coco.md

```diff
@@ -21,51 +21,53 @@ MobileNetV1 FPN is used for object detection. For details, see [paper](https://a
 
 ## Input
 
-### Original model
+### Original Model
 
-Image, name - `image_tensor`, shape - [1x640x640x3], format [BxHxWxC],
+Image, name: `image_tensor`, shape: [1x640x640x3], format: [BxHxWxC],
    where:
 
     - B - batch size
     - H - image height
     - W - image width
     - C - number of channels
 
-   Expected color order - RGB.
+   Expected color order: RGB.
 
-### Converted model
+### Converted Model
 
-Image, name - `image_tensor`, shape - [1x3x640x640], format [BxCxHxW],
+Image, name: `image_tensor`, shape: [1x3x640x640], format: [BxCxHxW],
    where:
 
     - B - batch size
     - C - number of channels
     - H - image height
     - W - image width
 
-   Expected color order - BGR.
+   Expected color order: BGR.
 
 ## Output
 
-### Original model
+### Original Model
 
-1. Classifier, name - `detection_classes`, contains predicted bounding boxes classes in range [1, 91]. The model was trained on Microsoft\* COCO dataset version with 90 categories of object, 0 class is for background.
-2. Probability, name - `detection_scores`, contains probability of detected bounding boxes.
-3. Detection box, name - `detection_boxes`, contains detection boxes coordinates in format `[y_min, x_min, y_max, x_max]`, where (`x_min`, `y_min`)  are coordinates top left corner, (`x_max`, `y_max`) are coordinates right bottom corner. Coordinates are rescaled to input image size.
-4. Detections number, name - `num_detections`, contains the number of predicted detection boxes.
+- Classifier, name: `detection_classes`. Contains predicted bounding-boxes classes in range [1, 91]. The model was trained on Microsoft\* COCO dataset version with 90 categories of object, 0 class is for background.
```

Comment:
Here numbered list was replaced with bulleted, but in some cases wasn't. Please explain why.